app/controllers/concerns/validators_controller_helper.rb

```diff
@@ -38,6 +38,17 @@ def create_json_result(validator, with_history = false)
     hash
   end
 
+  def index_csv_headers(with_history)
+    headers = Validator::FIELDS_FOR_API + 
+              ValidatorScoreV1::ATTRIBUTES_FOR_BUILDER + 
+              DataCenter::FIELDS_FOR_CSV + 
+              [:data_center_host] + 
+              [:vote_account] +
+              ValidatorHistory::FIELDS_FOR_CSV
```

Comment:
If the queue isn't matter this way is more readable.

```suggestion
    headers = Validator::FIELDS_FOR_API +
              ValidatorScoreV1::ATTRIBUTES_FOR_BUILDER +
              DataCenter::FIELDS_FOR_CSV +
              ValidatorHistory::FIELDS_FOR_CSV +
              [:data_center_host] +
              [:vote_account]
```